torch_sim/models/orb.py

```diff
@@ -416,7 +416,7 @@ def forward(self, state: ts.SimState | StateDict) -> dict[str, torch.Tensor]:
             if not model_has_direct_heads and prop == "stress":
                 continue
             _property = "energy" if prop == "free_energy" else prop
-            results[prop] = predictions[_property].squeeze()
+            results[prop] = predictions[_property]
```

Comment:
It'd be helpful to understand why the current tests aren't capturing this issue. If you could write a test that fails without this change that'd be really helpful.